.stylelintrc

```diff
@@ -1,8 +1,6 @@
 {
-  "extends": "stylelint-config-standard",
-  "plugins": [
-    "stylelint-order"
-  ],
+  "extends": ["stylelint-config-standard", "stylelint-prettier/recommended"],
+  "plugins": ["stylelint-order", "stylelint-scss"],
```

Comment:
You should also disable "at-rule-no-unknown" and enable "scss/at-rule-no-unknown".

Reply:
@bartaz done